api/main.py

```diff
@@ -31,7 +31,7 @@ async def ping():
     return "Hello, I am alive"
 
 def read_file_as_image(data) -> np.ndarray:
-    image = np.array(Image.open(BytesIO(data)))
+    image = np.array(Image.open(BytesIO(data)).resize(256, 256))
```

Comment:
`image = np.array(Image.open(BytesIO(data)).resize((256,256)))`

Reply:
IDK why but when I tried, resize worked only with double brackets around 256,256

Reply:
.resize() accepts a tuple. Therefore we need to provide double brackets. One for function call and one for the tuple.
Example:
imsize = (256,256)
now call the function with following
im.resize(imsize) # here imsize already has brackets in previous line. So its good to separate the code for better understanding.

Reply:
Yes, @jainharshit3107 You have to call resize method the image variable in the main.py file in api folder as @JKS46 showed.